docs/operator-guides/upgrading-airbyte.md

```diff
@@ -30,6 +30,10 @@ If you are running [Airbyte on Kubernetes](../deploying-airbyte/on-kubernetes.md
    docker-compose up
    ```
 
+{% hint style="info" %}
+If you did not start Airbyte from the root of the Airbyte monorepo, you will need to still need to manually remove these now orphaned Airbyte configurations. You can do this with `docker volume rm $(docker volume ls -q | grep airbyte)`.
```

Comment:
Right now, we don't have a section about how to reset an Airbyte instance to scratch anymore...

This command is equivalent to `docker-compose down -v` but it is more "global". Which means, it deletes all previous configuration of Airbyte (this one, regardless of where they were created from). There used to be a warning about this in the docs too.

Maybe @subodh1810 has more input on how to phrase this hint back better regarding to the automatic update?

Reply:
Yeah I understand what it does, I think I just need to be more clear that it’s removing all of the user’s data. Maybe I can add its own section instead of having just a hint - I just don’t want to make it seem like a normal workflow, since it shouldn’t be anymore.